jest.config.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
```

Comment:
migrated from the .json file, because an axios upgrade is required (to access the exported `AxiosError`), ...but of course doing that means a tiny jest change. jest by default resolves ESM, and axios exports ESM now. ...were testing in CJS though, so we need node to resolve axios as it natively would rather than letting jest's resolver do it. thus, i've moved the config to the .js config file format from the .json file format